### PR TITLE
whitespace cleanup

### PR DESCRIPTION
**Pull request by briskt**
### Changed
- Used PhpStorm "Reformat Code" to clean up whitespace.

### CODE REVIEW

File: features/bootstrap/LoginContext.php [hunk 1]
```diff
@@ -574,15 +575,16 @@ public function theIpAddressShouldHaveAFailedLoginAttempt($ipAddress)
     public function secondsAgoThatUsernameHadMoreFailedLoginsThanTheLimit(
         $numSeconds,
         $numFailuresBeyondLimit
-    ) {
+    )
+    {
```
Comment (mtompset) [inline features/bootstrap/LoginContext.php:579]: This is weird.

File: features/fakes/FakeIdBrokerClient.php [hunk 2]
```diff
@@ -25,16 +26,17 @@ class FakeIdBrokerClient
     public function __construct(
         string $baseUri,
         string $accessToken,
-        array $config = []
-    ) {
+        array  $config = []
+    )
+    {
```
Comment (mtompset) [inline features/fakes/FakeIdBrokerClient.php:31]: This is weird.

File: modules/material/themes/material/core/loginuserpass.php [hunk 3]
```diff
@@ -21,27 +21,28 @@ function preventDefault(event) {
...
-    <script src='https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoad&render=explicit' async defer></script>
-
-    <script>
-        function submitForm() {
-            document.querySelector('form').submit();
-        }
+    if (!empty($siteKey)) {
+        ?>
+        <script src='https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoad&render=explicit' async
+                defer></script>
```
Comment (mtompset) [inline modules/material/themes/material/core/loginuserpass.php:27]: I think the async probably should have moved with the defer for better wrapping here.

File: modules/material/themes/material/core/loginuserpass.php [hunk 4]
```diff
@@ -82,16 +83,18 @@ function onRecaptchaLoad() {
                         <?php
                         $username = htmlentities($this->data['username'] ?? null);
                         ?>
-                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black" value="<?= $username ?>"
-                               <?= empty($username) ? 'autofocus' : '' ?> id="username"/>
+                        <input type="text" required name="username" class="mdl-textfield__input mdl-color-text--black"
+                               value="<?= $username ?>"
```
Comment (mtompset) [inline modules/material/themes/material/core/loginuserpass.php:87]: I would have swapped the order of class and value so line wrapping would be nicer.

File: modules/material/themes/material/core/loginuserpass.php [hunk 5]
```diff
@@ -82,16 +83,18 @@ function onRecaptchaLoad() {
...
                     </div>
 
                     <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
                         <label for="password" class="mdl-textfield__label">
                             <?= $this->t('{material:login:label_password}') ?>
                         </label>
 
-                        <input type="password" required name="password" class="mdl-textfield__input mdl-color-text--black" <?= ! empty($username) ? 'autofocus' : '' ?>
+                        <input type="password" required name="password"
+                               class="mdl-textfield__input mdl-color-text--black" <?= !empty($username) ? 'autofocus' : '' ?>
```
Comment (mtompset) [inline modules/material/themes/material/core/loginuserpass.php:97]: Ugly line wrap could have been avoided by putting the <?= portion on the next line.

File: modules/material/themes/material/default/header.twig [hunk 6]
```diff
@@ -10,14 +10,18 @@
   <script async src="https://www.googletagmanager.com/gtag/js?id={{ trackingId }}"></script>
   <script>
     window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
+
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+
```
Comment (mtompset) [inline modules/material/themes/material/default/header.twig:17]: Pretty! :)

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 7]
```diff
@@ -437,15 +437,16 @@ protected static function isHeadedToMfaSetupUrl($state, $mfaSetupUrl)
...
+        string|array    $mfaSubmission,
+        array           $state,
+        bool            $rememberMe,
         LoggerInterface $logger,
-        string $mfaType,
-        string $rpOrigin
-    ): string {
+        string          $mfaType,
+        string          $rpOrigin
+    ): string
```
Comment (mtompset) [inline modules/mfa/src/Auth/Process/Mfa.php:448]: This is weird to me.

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 8]
```diff
@@ -706,11 +707,12 @@ protected function redirectToMfaPrompt(array &$state, string $employeeId, array
     public static function isRememberMeCookieValid(
         string $cookieHash,
         string $expireDate,
-        array $mfaOptions,
-        array $state
-    ): bool {
+        array  $mfaOptions,
+        array  $state
+    ): bool
+    {
```
Comment (mtompset) [inline modules/mfa/src/Auth/Process/Mfa.php:713]: This is weird to me.

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 9]
```diff
@@ -732,9 +734,10 @@ public static function isRememberMeCookieValid(
     public static function generateRememberMeCookieString(
         string $rememberSecret,
         string $employeeId,
-        int $expireDate,
-        array $mfaOptions
-    ): string {
+        int    $expireDate,
+        array  $mfaOptions
+    ): string
+    {
```
Comment (mtompset) [inline modules/mfa/src/Auth/Process/Mfa.php:740]: This is weird to me.

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 10]
```diff
@@ -758,20 +761,21 @@ public static function generateRememberMeCookieString(
...
      */
     protected static function redirectToLowOnBackupCodesNag(
-        array &$state,
+        array  &$state,
         string $employeeId,
-        int $numBackupCodesRemaining
-    ): void {
+        int    $numBackupCodesRemaining
+    ): void
+    {
```
Comment (mtompset) [inline modules/mfa/src/Auth/Process/Mfa.php:768]: This is weird to me.

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 11]
```diff
@@ -801,9 +805,10 @@ protected static function redirectToOutOfBackupCodesMessage(array &$state, strin
      */
     public static function setRememberMeCookies(
         string $employeeId,
-        array $mfaOptions,
+        array  $mfaOptions,
         string $rememberDuration = '+30 days'
-    ): void {
+    ): void
+    {
```
Comment (mtompset) [inline modules/mfa/src/Auth/Process/Mfa.php:811]: This is weird to me.

File: modules/silauth/src/Auth/Source/auth/Authenticator.php [hunk 12]
```diff
@@ -43,23 +44,24 @@ public function __construct(
         Captcha         $captcha,
         IdBroker        $idBroker,
         LoggerInterface $logger
-    ) {
+    )
+    {
```
Comment (mtompset) [inline modules/silauth/src/Auth/Source/auth/Authenticator.php:48]: This is weird to me.

File: modules/silauth/src/Auth/Source/auth/IdBroker.php [hunk 13]
```diff
@@ -31,13 +32,14 @@ class IdBroker
...
         LoggerInterface $logger,
-        string $idpDomainName,
-        array $trustedIpRanges,
-        bool $assertValidIp = true
-    ) {
+        string          $idpDomainName,
+        array           $trustedIpRanges,
+        bool            $assertValidIp = true
+    )
+    {
```
Comment (mtompset) [inline modules/silauth/src/Auth/Source/auth/IdBroker.php:42]: This is weird to me.

File: modules/silauth/src/Auth/Source/tests/fakes/FakeIdBroker.php [hunk 14]
```diff
@@ -11,11 +12,12 @@
...
-        string $baseUri,
-        string $accessToken,
+        string          $baseUri,
+        string          $accessToken,
         LoggerInterface $logger,
-        string $idpDomainName = 'fake.example.com'
-    ) {
+        string          $idpDomainName = 'fake.example.com'
+    )
+    {
```
Comment (mtompset) [inline modules/silauth/src/Auth/Source/tests/fakes/FakeIdBroker.php:20]: This is weird to me.

File: tests/MetadataTest.php [hunk 15]
```diff
@@ -107,11 +108,10 @@ public function testIDPRemoteMetadataBadSPList()
     }
```
Comment (mtompset) [inline tests/MetadataTest.php:110]: Probably could have removed another line.

**Comment by mtompset** [comment]
Most of my comments are likely related to:
"When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them." (https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments)
"When you have a return type declaration present, there MUST be one space after the colon followed by the type declaration. The colon and declaration MUST be on the same line as the argument list closing parenthesis with no spaces between the two characters. ... ): string {"

**Review by hobbitronics — APPROVED** [review]
Looks like you've resolved the comments

**Comment by briskt** [comment]
> Most of my comments are likely related to: "When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them." (https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments) "When you have a return type declaration present, there MUST be one space after the colon followed by the type declaration. The colon and declaration MUST be on the same line as the argument list closing parenthesis with no spaces between the two characters. ... ): string {"

Thanks for bringing this to my attention. I naively assumed that the default phpStorm settings would be PSR-12 compliant.

**Review by forevermatt — APPROVED** [review]
Thanks for doing this

**Review by jason-jackson — APPROVED** [review]

**Pull request merged**